apps/dot-voting/package.json

```diff
@@ -63,15 +65,24 @@
     "eslint-plugin-promise": "^4.1.1",
     "eslint-plugin-react": "^7.13.0",
     "eslint-plugin-standard": "^4.0.0",
+    "eth-gas-reporter": "0.1.12",
+    "ethereumjs-abi": "0.6.5",
+    "ethlint": "^1.2.4",
```

Comment:
@ottodevs Quazia and I ran into problems with Ethlint in CI for Discussions, and it looks like it's also causing problems in this branch. Can we delay these updates for another time and continue with solium for a bit longer? Also, is it okay if we continue to use the same soliumrc configuration we've been using in dev? I'd prefer to discuss style changes, especially of this magnitude